HelloFlask/views.py

```diff
@@ -1,31 +1,44 @@
-from flask import Flask
-from flask import render_template
+from datetime import datetime
+from re import match
```

Comment:
usually people just `import re` and then use `re.match()` - this also avoids shadowing when people assign `match = ...` to the actual match object

Reply:
Makes sense...I'm still unclear when it's good to import a module as a whole vs. importing the part you need. :)